src/services/helpdesk/index.js

```diff
@@ -2,11 +2,23 @@ const service = require('feathers-mongoose');
 const problemModel = require('./model');
 const hooks = require('./hooks');
 const logger = require('../../logger');
-const { BODYPARSER_JSON_LIMIT, MAXIMUM_ALLOWABLE_TOTAL_ATTACHMENTS_SIZE_BYTE } = require('../../../config/globals');
+const {} = require('../../../config/globals');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `./model`](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)